Description: [superformat] Make --verify_later an option without argument
 The superformat option --verify_later requires an argument, but the
 variable verify_later is always used as a switch (0/1). This patch
 suppresses the required argument.
Author: Matteo Cypriani <mcy@lm7.fr>
Bug-Debian: http://bugs.debian.org/331488
Last-Update: 2021-01-30
--- a/src/superformat.c
+++ b/src/superformat.c
@@ -562,7 +562,7 @@
 		(void *) &dosverify,
 		"verify disk using mbadblocks" },
 
-	{ 'V', "verify_later", 1, EO_TYPE_BYTE, 1, 0,
+	{ 'V', "verify_later", 0, EO_TYPE_BYTE, 1, 0,
 		(void *) &verify_later,
 		"verify floppy after all formatting is done" },
 
